pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -76,7 +80,7 @@ for arg; do
     else
         case $argfun in
             set_*)
-                var=$(echo $argfun | sed 's,^set_,,')
+                var=${argfun/#set_/}
```

Comment:
It's a bit weird but `#` acts as the `^` character in normal regexp. At least that's what I've tested.